client/src/components/Today/ChangeDayButton.tsx

```diff
@@ -14,6 +14,7 @@ export default function ChangeDayButton({
 
 	return (
 		<S.ChangeDayButton
+			color={"darkBlue"}
```

Comment:
Some buttons use `$color`, others `color`. Use `$color` everywhere so we don't accidentally override an actual color prop.